packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -1,15 +1,17 @@
-import {EventEmitter} from 'events';
 import {promises as fs} from 'fs';
 import path from 'path';
 import {SnowpackBuildMap, SnowpackPlugin} from '../types/snowpack';
 import {getEncodingType, getExt} from '../util';
 import {validatePluginLoadResult} from '../config';
+import createLogger from '../logger';
+
+const logger = createLogger();
```

Comment:
In each module, I wanted to have a separate instance, hence the function.

Since we now have the concept of “silent mode,” I wanted `build()` to be able to be “noisy” while also being able to run `install()` in “silent mode.”

This will be important for the dev server, where we can run commands and choose whether or not they’ll display information.

I wanted to be able, for example, for

Reply:
Update: came up with a different solution that’s simpler: creating multiple loggers with different `name`s